src/components/entries/Tooltip.js

```diff
@@ -0,0 +1,123 @@
+import { useRef } from 'preact/hooks';
+import {
+  useEffect,
+  useState
+} from 'react';
+
+/**
+ * @param {Object} props
+ * @param {String} props.labelId
+ * @param {String} props.value
+ */
+export default function Tooltip(props) {
+  const {
+    value,
+    labelId
+  } = props;
+
+  const [ show, setShow ] = useState(false);
+
+  const wrapperRef = useRef(null);
+  const tooltipRef = useRef(null);
+
+  const showTooltip = async () => {
+
+    setTimeout(() => {
+      setShow(true);
+    }, 200);
+  };
+  const hideTooltip = () => setShow(false);
+
+  const isTooltipHovered = ({ x, y }) => {
+    const tooltip = tooltipRef.current;
+    const wrapper = wrapperRef.current;
+
+    return tooltip && (
+      inBounds(x, y, wrapper.getBoundingClientRect()) ||
+      inBounds(x, y, tooltip.getBoundingClientRect())
+    );
+  };
+
+  useEffect(() => {
+    const { current } = wrapperRef;
+
+    if (!current) {
+      return;
+    }
+
+    const hideHoveredTooltip = (e) => {
+      if (!isTooltipHovered({ x: e.x, y: e.y })) {
+        hideTooltip();
+      }
+    };
+
+    const hideFocusedTooltip = (e) => {
+      const { relatedTarget } = e;
+      const isTooltipChild = (el) => el && el.closest('.bio-properties-panel-tooltip');
+
+      if (!isTooltipChild(relatedTarget)) {
+        hideTooltip();
+      }
+    };
+
+    const hideTooltipViaEscape = (e) => {
+      e.code === 'Escape' && hideTooltip();
+    };
+
+    current.addEventListener('mouseover', showTooltip);
+    current.addEventListener('focusin', showTooltip);
+    current.addEventListener('keydown', hideTooltipViaEscape);
+    document.addEventListener('focusout', hideFocusedTooltip);
+    document.addEventListener('mousemove', hideHoveredTooltip);
```

Comment:
What's the reason we use DOM elements and not preact wrappers (onMouseMove etc.)?

Reply:
I think I know why we do it for `document` listeners but not for the element.

Reply:
Also: `mouseover` vs. `mouseenter`, `focusout` vs. `blur`?

Reply:
`focusout` over `blur` is because `focusout` bubbles, while `blur` does not. So we can add it the document and listen to any `focusout` events.

`mouseover` vs. `mouseenter` there wasn't really a reason. I changed it and works just fine